superset-frontend/src/components/ErrorMessage/ErrorAlert.tsx

```diff
@@ -40,6 +40,11 @@ const ErrorAlertDiv = styled.div<{ level: ErrorLevel }>`
     justify-content: space-between;
   }
 
+  .monospace {
+    font-family: monospace;
```

Comment:
If you were to use Emotion, you could make sure we use Fira Code here instead of the browser's go-to monospace font. 
`${({ theme }) => theme.typography.families.monospace}`

Reply:
ooouh, fancy!